src/fal/planner/executor.py

```diff
@@ -0,0 +1,101 @@
+from __future__ import annotations
```

Comment:
ATL: what does this do?

Reply:
It postpones the execution of annotations. Pretty much responsible for bringing more flexibility to annotations when needed (when I was using `list[x]` syntax or `x | y`, it was allowing these to run in 3.7 even though they are not implemented at that version). Right now, it doesn't add any value, removing it!